### PR TITLE
Add flowMapping context to TP

### PR DESCRIPTION
**Pull request by gasserluc**
I need to add flowMapping to TP context in order to start clean for Antibes
Since TP context also contains the bridge mapping context I needed to align it on vocab.egm.io to not break everything

### CODE REVIEW

**Review by junjou — APPROVED** [review]
Nice but with this we will have to update TP because I know somewhere we are using a hard coded context uri just for getting bridges entities

File: mapping/jsonld-contexts/mapping.jsonld [hunk 1]
```diff
@@ -1,23 +1,23 @@
 {
 	"@context": {
-		"Mapping": "https://ontology.eglobalmark.com/Mapping",
```
Comment (bobeal) [inline mapping/jsonld-contexts/mapping.jsonld:3]: guess it will probably break the MQTT bridge. do we really want to add this burden?

Comment (gasserluc) [reply mapping/jsonld-contexts/mapping.jsonld:3]: No we don't :)
If I leave Mapping and Bridge to the old expanded will it be fine ? (they don't have any impact on the flow mapping)

Comment (gasserluc) [reply mapping/jsonld-contexts/mapping.jsonld:3]: I really only need to change "attributesToMap", "attribute", "context" and "hasTargetEntity"

Comment (bobeal) [reply mapping/jsonld-contexts/mapping.jsonld:3]: good news (for you), it should not have any impact on MQTT Bridge

Comment (gasserluc) [reply mapping/jsonld-contexts/mapping.jsonld:3]: No impact if I revert Mapping and Bridge or can I go like that ?

Comment (bobeal) [reply mapping/jsonld-contexts/mapping.jsonld:3]: the full modification

**Pull request merged**